project/frontend/src/pages/MatchResultsPage.js

```diff
@@ -20,14 +20,33 @@ function MatchResultsPage(props) {
   useEffect(() => {
     getRecommendation(formState, (result, err) => {
       if (err) {
+        console.log('error');
```

Comment:
these error and no error logs don't seem necessary?

Reply:
Forgot to remove those. Thanks for catching that!